app/controllers/course_reports_controller.rb

```
@@ -0,0 +1,132 @@
+      [assignment.id.to_s, participant ? build_assignment_cell(assignment, participant) : nil]
+    end
+  end
+
+  def build_assignment_cell(assignment, participant)
+    team = participant.team
+
+    {
+      participant_id: participant.id,
+      peer_grade: team&.aggregate_review_grade,
+      instructor_grade: team&.grade_for_submission,
+      avg_teammate_score: participant.aggregate_teammate_review_grade(teammate_review_maps_for(assignment, participant)),
```

RuboCop (1.86.1)

[convention] Line is too long. [121/120]

(Layout/LineLength)